docs/integrations/destinations/mssql-migrations.md

```diff
@@ -0,0 +1,65 @@
+# MSSQL Migration Guide
+
+## Upgrading to 1.0.0
+
+This version removes the option to use "normalization" with MS SQL Server. It also changes
+the schema and database of Airbyte's "raw" tables to be compatible with the new
+[Destinations V2](https://docs.airbyte.com/release_notes/upgrading_to_destinations_v2/#what-is-destinations-v2)
+format. These changes will likely require updates to downstream dbt / SQL models. After this update,
+Airbyte will only produce the ‘raw’ v2 tables, which store all content in JSON. These changes remove
+the ability to do deduplicated syncs with MS SQL Server.
+If you are interested in the MS SQL Server destination gaining the full features
+of Destinations V2 (including final tables), click [[https://github.com/airbytehq/airbyte/discussions/37010]]
```

Comment:
line 10/11: these should also be changed to either MSSQL Server or Microsoft SQL Server